_includes/pbc/all/install-features/202304.0/install-glue-api/install-backend-api-glue-json-api-convention.md

```diff
@@ -0,0 +1,141 @@
+
```

Comment:
* We have to add the header file for the page by the following path:
* docs/scos/dev/feature-integration-guides/202304.0/install-backend-api-glue-json-api-convention.md